#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>

QT_BEGIN_NAMESPACE
namespace Ui { class MainWindow; }
QT_END_NAMESPACE

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    MainWindow(QWidget *parent = nullptr);
    ~MainWindow();
    
private slots:
    void on_pushButton_clicked();

    void on_Button2_clicked();

    void on_pushButton_2_clicked();
    void on_Button3_clicked();
    void on_Button4_clicked();
    void on_Button5_clicked();
    void on_Button6_clicked();
    void on_Button7_clicked();
    void on_Button8_clicked();
    void on_Button9_clicked();
    void on_Button10_clicked();
    void on_Button11_clicked();
    void on_Button12_clicked();
    void on_Button13_clicked();
    void on_Button14_clicked();
    void on_Button15_clicked();
    void on_Button16_clicked();
    void on_Button17_clicked();
    void on_Button18_clicked();
    void on_Button19_clicked();
    void on_Button20_clicked();
    void on_Button21_clicked();
    void on_Button22_clicked();
    void on_Button23_clicked();
    void on_Button24_clicked();
    void on_Button25_clicked();
    void on_Button26_clicked();
    void on_Button27_clicked();
    void on_Button28_clicked();
    void on_Button29_clicked();
    void on_Button30_clicked();
    void on_Button31_clicked();
    void on_Button32_clicked();
    void on_Button33_clicked();
    void on_Button34_clicked();
    void on_Button35_clicked();
    void on_Button36_clicked();
    void on_Button37_clicked();
    void on_Button38_clicked();
    void on_Button39_clicked();
    void on_Button40_clicked();
    void on_Button41_clicked();
    void on_Button42_clicked();
    void on_Button43_clicked();
    void on_Button44_clicked();
    void on_Button45_clicked();
    void on_Button46_clicked();
    void on_Button47_clicked();
    void on_Button48_clicked();
    void on_Button49_clicked();
    void on_Button50_clicked();
    void on_Button51_clicked();
    void on_Button52_clicked();
    void on_Button53_clicked();
    void on_Button54_clicked();
    void on_Button55_clicked();
    void on_Button56_clicked();
    void on_Button57_clicked();
    void on_Button58_clicked();
    void on_Button59_clicked();
    void on_Button60_clicked();

    void on_pushButton_3_clicked();

private:
    Ui::MainWindow *ui;
};
#endif // MAINWINDOW_H

